internal/utils/validation/cd.go

```diff
@@ -16,53 +16,15 @@ package validation
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"slices"
 	"strings"
 
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/log"
 
 	kcmv1 "github.com/K0rdent/kcm/api/v1beta1"
 )
 
-// ClusterDeployCrossNamespaceServicesRefs validates that the service and templates references of the given [github.com/K0rdent/kcm/api/v1beta1.ClusterDeployment]
-// reference all objects only in the obj's namespace.
-func ClusterDeployCrossNamespaceServicesRefs(ctx context.Context, cd *kcmv1.ClusterDeployment) (errs error) {
```

Comment:
Why remove this validation? The ClusterDeployment is namespace scoped so should refer to only ServiceTemplates within its namespace right? 

The AccessManagement object is used to propagate Cluster/ServiceTemplates across namespaces [as mentioned](https://docs.k0rdent.io/v1.1.1/admin/?h=accessmanagement#the-process:~:text=kcm%2Dcontroller%2Dmanager%20generates%20an%20empty%20AccessManagement%20object) in the k0rdent docs. The removal of this validation would break this model.